lib/index.d.ts

```diff
@@ -0,0 +1,104 @@
+// Type definitions for panel 0.10.0
+// Project: panel
+// Definitions by: Mixpanel (https://mixpanel.com)
+import {VNode} from 'snabbdom';
+import {WebComponent} from 'webcomponent';
+
+export class Component extends WebComponent {
+    /**
+     *  Defines the state of the component, including all the properties required for rendering.
+     */
+    state: any;
+    /**
+     *  Defines standard component configuration.
+     */    
+    config: Component.ComponentConfigOptions;
+    /**
+     *  Template helper functions defined in config object, and exposed to template code as $helpers.
+     *  This getter uses the component's internal config cache.
+     */
+    helpers: any;
+    /**
+     * Called when an instance of the Component is attached to the DOM.
+     */
+     attachedCallback(): void;
```

Comment:
fixing the indentation -- I think in light of how close we are to moving over to Custom Elements v1, I think we should hold off on these types until we upgrade. will find another place to add types for the HTMLElement methods, agreed that they don't really belong here